lab-michael/lib/storage.js

```diff
@@ -0,0 +1,55 @@
+
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'), {suffix: 'Prom'});
+
+const storage = {};
+
+exports.fetchAll = function(recipe) {
```

Comment:
You're using the exports object here, and you should be explicit about assigning this to the module.exports object, or just use module.exports explicitly.